otx/cli/manager/config_manager.py

```diff
@@ -257,11 +259,15 @@ def _get_task_type_from_data_format(self, data_format: str) -> str:
                 return task_key
         raise ConfigValueError(f"Can't find proper task. we are not support {data_format} format, yet.")
 
-    def auto_split_data(self, data_roots: str, task: str):
+    def auto_split_data(self, data_roots: str, task: str, train_ann_file: Optional[str] = None):
         """Automatically Split train data --> train/val dataset."""
         self.data_format = self.dataset_manager.get_data_format(data_roots)
         dataset = self.dataset_manager.import_dataset(data_root=data_roots, data_format=self.data_format)
         train_dataset = self.dataset_manager.get_train_dataset(dataset)
```

Comment:
Maybe we could move line 266-270 to the line 280 else statement? It looks to me like train_dataset is not used in any other case.

Reply:
Oh, my bad, it's used on line 277.